storagenode/orders/service.go

```diff
@@ -149,6 +150,11 @@ func (service *Service) sendOrders(ctx context.Context) (err error) {
 
 	const batchSize = 1000
 
+	jitter := time.Duration(rand.Int63n(int64(time.Second * 300)))
```

Comment:
This probably needs to be configurable. Also, we should disable this for running tests.

Reply:
i.e. see how `nodestats` does it.

Reply:
Done